### PR TITLE
Feat/header

### PR DESCRIPTION
**Pull request by Quannichan**

### CODE REVIEW

File: src/components/header_compo.html [hunk 1]
```diff
@@ -1 +1,15 @@
+<div class="header_component">
```
Comment (Lanng21) [inline src/components/header_compo.html:1]: Can u use senmatic html?

File: src/components/header_compo.html [hunk 2]
```diff
@@ -1 +1,15 @@
+<div class="header_component">
+    <div class="wrap_components">
+        <div class="compo1">
+            <p class="head_header">See The Unmatched Beauty Of The Great Lakes</p>
+        </div>
+        <div class="wrap_video_header">
+            <iframe class="video_header" src="https://www.youtube.com/embed/tgbNymZ7vqY">
+            </iframe>
+        </div>
+        <div class="wrap_arrow">
```
Comment (Lanng21) [inline src/components/header_compo.html]: use figure tag instead

File: src/components/menu.html [hunk 3]
```diff
@@ -1 +1,20 @@
-
+<div class="wrap_menu">
+    <div class="wrap_trademark">
+            <p class="trademark">HOFMANN</p>
+    </div>
+<div class="wrap_list1_menu">
+    <ul class="list1_menu">
```
Comment (Lanng21) [inline src/components/menu.html:7]: move to list component and use each tag

File: src/components/menu.html [hunk 4]
```diff
@@ -1 +1,20 @@
...
+        <li class="list1">HOTELS</li>
+        <li class="list1">AIRLINES</li>
+        <li class="list1">VACATIONS</li>
+        <li class="list1">FIND MORE</li>
+    </ul>
+</div>
+<div class="wrap_list2_menu">
+    <ul class="list2_menu">
```
Comment (Lanng21) [inline src/components/menu.html:20]: same

File: src/components/menu.html [hunk 5]
```diff
@@ -1 +1,20 @@
...
+        <li class="lits2"><img class="img_lits2" src="../images/twitter_white.png" alt=""></li>
+        <li class="lits2"><img class="img_lits2" src="../images/facebook_white.png" alt=""></li>
+        <li class="lits2"><img class="img_lits2" src="../images/instagram_white.png" alt=""></li>
+    </ul>
+</div>
+</div>
```
Comment (Lanng21) [inline src/components/menu.html]: end of line

File: src/layouts/header.html [hunk 6]
```diff
@@ -1 +1,8 @@
-
+<header class="wrap_header">
+    <div class="image_header">
+    </div>
+    <!-- include menu components -->
+    <include src="./components/menu.html" ></include>
+    <!-- include header components -->
+    <include src="./components/header_compo.html" ></include>
+</header>
```
Comment (Lanng21) [inline src/layouts/header.html]: same

File: src/stylesheet/Main.scss [hunk 7]
```diff
@@ -1 +1,110 @@
-
+@import "./backgrounds";
+*{
```
Comment (Lanng21) [inline src/stylesheet/Main.scss]: move to base folder in stylesheets

File: src/stylesheet/Main.scss [hunk 8]
```diff
@@ -1 +1,110 @@
...
+body{
+    font-family: Roboto;
+}
+.wrap_header{
+    display: block;
+}
+.image_header{
+    z-index: -1;
+    width: 100%;
+    height: 768px;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss]: dont fixed w h

File: src/stylesheet/Main.scss [hunk 9]
```diff
@@ -1 +1,110 @@
...
+    padding: 3%;
+    justify-content: space-between;
+    align-items: center;
+}
+.wrap_trademark{
+    display: flex;
+    justify-content: center;
+}
+.trademark{
+    color: #EBCB80;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss]: move to variable

File: src/stylesheet/Main.scss [hunk 10]
```diff
@@ -1 +1,110 @@
...
+}
+.list1_menu{
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    list-style: none;
+    column-gap: 30px;
+}
+.list1{
+    color: #FFF;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss]: same

File: src/stylesheet/Main.scss [hunk 11]
```diff
@@ -1 +1,110 @@
...
+}
+.list2_menu{
+    display: flex;
+    list-style: none;
+    justify-content: space-evenly;
+    align-items: center;
+    column-gap: 30px;
+}
+.img_lits2{
+    width: 25px;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss]: dont fixed w h for img

File: src/stylesheet/Main.scss [hunk 12]
```diff
@@ -1 +1,110 @@
...
+}
+.wrap_components{
+    display: block;
+    text-align: center;
+}
+.compo1{
+    margin-bottom: 30px;
+}
+.head_header{
+    color: #FFF;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss]: same

File: src/stylesheet/Main.scss [hunk 13]
```diff
@@ -1 +1,110 @@
...
+    text-align: center;
+    font-family: Roboto;
+    font-size: 44px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 48px; /* 109.091% */
+    letter-spacing: -1.6px;
+    width: 550px;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss]: same

File: src/stylesheet/Main.scss [hunk 14]
```diff
@@ -1 +1,110 @@
...
+}
+.wrap_video_header{
+    display: flex;
+    justify-content: center;
+    margin-bottom: 100px;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss]: margin 100px too large

File: src/stylesheet/Main.scss [hunk 15]
```diff
@@ -1 +1,110 @@
...
+}
+.video_header{
+    border: transparent 0px solid;
+    width: 555px;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss]: same

File: src/stylesheet/_backgrounds.scss [hunk 16]
```diff
@@ -1 +1 @@
-
+$back_ground_header : url("../images/background_header.png");
```
Comment (Lanng21) [inline src/stylesheet/_backgrounds.scss]: end of line

**Review by Lanng21 — COMMENTED** [review]
pls check comment and resolve and check alt img, git convention...

File: .gitignore [hunk 17]
```diff
@@ -1 +1,3 @@
-/node_modules
\ No newline at end of file
+/node_modules
+/.parcel-cache
+/dist
```
Comment (Lanng21) [inline .gitignore]: end of line

File: src/stylesheet/Main.scss [hunk 18]
```diff
@@ -1 +1,110 @@
...
+    margin-bottom: 30px;
+}
+.head_header{
+    color: #FFF;
+    text-align: center;
+    font-family: Roboto;
+    font-size: 44px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 48px; /* 109.091% */
```
Comment (Lanng21) [inline src/stylesheet/Main.scss]: delete cmt

File: src/components/menu.html (hunk 5 above)
Comment (Quannichan) [reply src/components/menu.html]: i dont know how to fix that

File: src/stylesheet/Main.scss [hunk 19]
```diff
@@ -1 +1,270 @@
...
+    .compo1{
+        margin-bottom: 7vh;
+    }
+    .head_header{
+        @include header_header_desktop;
+    }
+    .wrap_video_header{
+        display: flex;
+        justify-content: center;
+        margin-bottom: 12vh;
```
Comment (Lanng21) [inline src/stylesheet/Main.scss]: why u use vh here? u understand CSS units?

**Review by Lanng21 — COMMENTED** [review]
pls check CSS units? i see u use vh every where

**Review by Lanng21 — COMMENTED** [review]
Can u apply scss 7-1 partten?

**Review by Lanng21 — COMMENTED** [review]
Dont set w h for img. I think I've mentioned it many times but you still don't do it.

**Pull request closed without merging**